.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+##### SUMMARY
+<!--- Describe the change below, including rationale and design decisions -->
+
+<!--- HINT: Include "Fixes #nnn" if you are fixing an existing issue -->
+
+<!--- Please do not forget to include a changelog fragment:
+      https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment
```

Comment:
It might be intersting to point to one header level up, to pick on the beginning of that topic in that page. I do notice though, that it in that page the "test-only PR" is not explicitly listed as exempted from the need of creating a changelog fragment.
```suggestion
      https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-changelog-fragments
```